charts/gadget/templates/daemonset.yaml

```diff
@@ -184,18 +210,50 @@ spec:
       tolerations:
         {{- toYaml .Values.tolerations | nindent 8 }}
       volumes:
-        - name: host
+        # /bin is needed to find runc.
+        - name: bin
+          hostPath:
+            path: /bin
+        # /etc/os-release is needed by entrypoint to print information.
+        - name: etc-os-release
+          hostPath:
+            path: /etc/os-release
+        # /etc/nri is needed by entrypoint and cleanup to handle NRI hooks.
+        - name: etc-nri
           hostPath:
-            path: /
+            path: /etc/nri
```

Comment:
The gadget pod would fail to start if `/etc/nri` does not exist on the host. I think we unfortunately have to mount the full /etc to avoid this issue.

Ditto for some other paths:
- `/etc/containers/oci/`
-  `/opt/hooks/oci`
- `/usr/share/containers/oci/`
- etc.